examples/bpi-monolith-test-suite/bpi/components/zkcircuits/ordercircuit/order.sol

```diff
@@ -0,0 +1,231 @@
+
```

Comment:
same question for all gnark-generated files: which files do we need to add to repo?